bridgehead

```diff
@@ -41,6 +41,7 @@ case "$PROJECT" in
 		;;
 esac
 
+# TODO: Please add proper documentation for variable priorities (1. secrets, 2. vars, 3. PROJECT.local.conf, 4. PROJECT.conf, 5. ???
```

Comment:
Remove Todo

Reply:
Added to https://github.com/samply/bridgehead/issues/166

Reply:
Removed